### ISSUE TITLE
LHM doesn't copy data when only 1 row and ID isn't 1

### ISSUE THREAD

**Issue by aselder** [soundcloud/lhm#149]
In chunker.rb, the following while loop exists:

```
      while @next_to_insert < @limit || (@next_to_insert == 1 && @start == 1)
        stride = @throttler.stride
        affected_rows = @connection.update(copy(bottom, top(stride)))

        if @throttler && affected_rows > 0
          @throttler.run
        end

        @printer.notify(bottom, @limit)
        @next_to_insert = top(stride) + 1
      end
```

when `@next_to_insert` starts as the `min(id)` and `@limit` is the `max(id)`. So if the table contains a single row with an id of `2`, then `@next_to_insert` is equal to `@limit` and the first half of the `||` evaluates to false and the values are not equal to `1` so the second half is also false, so the copy never happens.

**Comment by aselder** [comment]
I also ran this with 40001 records. After Lhm, the resulting table only have 40000 records.

**Comment by tdeo** [comment]
Hi @aselder,

I've got this issue opened for 3 years about the same issue: https://github.com/soundcloud/lhm/issues/126, and two PR to add test (https://github.com/soundcloud/lhm/pull/134) and fix the issue (https://github.com/soundcloud/lhm/pull/127). 

However, this repo seems to be unmaintained, so I was planning to open another issue to have it stated on the Readme

**Comment by grobie** [comment]
Fixed with #134
